labextension/vpython/README.md

```diff
@@ -22,6 +22,7 @@ For a development install (requires npm version 4 or later), do the following in
 npm install
 npm install --save @jupyterlab/notebook @jupyterlab/application @jupyterlab/apputils @jupyterlab/docregistry @phosphor/disposable script-loader
```

Comment:
```suggestion
```
I was able to omit this and still build the extension successfully.

Reply:
Incorporated this change in squash-commit f363918.